ecommerce_integrations/shopify/doctype/shopify_setting/shopify_setting.json

```diff
@@ -204,7 +204,7 @@
    "default": "0",
    "fieldname": "sync_sales_invoice",
    "fieldtype": "Check",
-   "label": "Import Sales Invoice from Shopify if Payment is marked"
+   "label": "Create Sales Invoice and Payment Entry when an order is marked as paid in Shopify"
```

Comment:
This is quite verbose for field labels. Also "marked as" is not the norm. Usually customers just pay and it's automated.

Reply:
Best labels IMO could be "Sync Deliveries from Shopify" and "Sync Sales Invoice from shopify"


Rest can be added in description (subtext below label)

Reply:
Yes I think keeping the label short and adding details as description is a good idea.

Reply:
> This is quite verbose for field labels. Also "marked as" is not the norm. Usually customers just pay and it's automated.

Also, this is true for web shops where the only sales channel is the online store (prepaid orders) but for cases where orders are also manually punched in Shopify (phone, mail B2B orders), the payment status has to be manually changed (marked) from "payment pending" to "paid" once it is settled (usually outside of Shopify's checkout system)